File: level1/문자열 내 p와 y의 개수/Solution.java
class Solution {
    boolean solution(String s) {
        boolean answer;
        char[] arr = s.toCharArray();
        int countp = 0;
        int county = 0;
        for (int i = 0; i < arr.length; i++) {
            if (arr[i] == 'y' || arr[i] == 'Y') {
                county++;
            } else if (arr[i] == 'p' || arr[i] == 'P')
                countp++;
        }
        if (county == countp) {
            answer = true;
        } else if (county + countp == 0) {
            answer = true;
        } else
            answer = false;
        return answer;
    }
}

File: level1/문자열 내 p와 y의 개수/Solution.js
function solution(s) {
    let count = 0;
    for (let i = 0; i < s.length; i++) {
        if (s.charAt(i) == 'p' || s.charAt(i) == 'P') count++;
        if (s.charAt(i) == 'y' || s.charAt(i) == 'Y') count--;
    }
    return (count == 0) ? true : false;
}
